packages/components/atoms/f-segmented-control/src/components/SegmentedControl.vue

```diff
@@ -0,0 +1,254 @@
+<template>
+    <div
+        :class="{
+            [$style['c-segmented-control']]: true,
+            [$style['c-segmented-control--large']]: size === 'large'
+        }"
+        data-test-id="segmented-control"
+        :aria-label="screenreaderLabel"
+        role="radiogroup"
+        tabindex="-1"
+        @keydown="onKeyDown"
+    >
+        <button
+            v-for="(option, index) in options"
+            :key="option.label"
+            :data-test-id="`segmented-control-option-${index}`"
+            type="button"
+            :class="{
+                [$style['c-segmented-control__option']]: true,
+                [$style['c-segmented-control__option--selected']]: option.label === selectedLabel
```

Comment:
Small naming thing (use camelCase for naming and dashes for children)

```suggestion
                [$style['c-segmentedControl-option']]: true,
                [$style['c-segmentedControl-option--selected']]: option.label === selectedLabel
```